Doc/library/os.rst

```diff
@@ -3780,6 +3780,174 @@ features:
    .. versionadded:: 3.10
 
 
+.. function:: timerfd_create(clockid, flags, /)
+
+   Create and return a timer file descriptor (*timerfd*).
+   The file descriptor supports :func:`read` with a buffer size of 8,
+   :func:`~select.select`, and :func:`~select.poll`.
+
+   .. seealso:: The :manpage:`timerfd_create(2)` man page.
+
+   *clockid* must be a valid :ref:`clock ID <time-clock-id-constants>`,
+   as defined in the :py:mod:`time` module.
+
+   The file descriptor's behaviour can be modified by specifying a *flags* value.
+   Any of the following variables may used, combined using bitwise OR
+   (the ``|`` operator):
+
+   - :const:`TFD_NONBLOCK`
+   - :const:`TFD_CLOEXEC`
+
+   If the timer file descriptor's counter is zero and
+   :const:`TFD_NONBLOCK` is not set as a flag, :func:`read` blocks.
+
+   The following example shows how to use a timer file descriptor
+   to execute a function twice a second:
```

Comment:
Would this run forever (until an exception)? This should be made explicit

Reply:
@AA-Turner 
Changed the sample code.